mod_reforged/hooks/skills/skill_container.nut

```diff
@@ -0,0 +1,9 @@
+::Reforged.HooksMod.hook("scripts/skills/skill_container", function(q) {
+	q.onDamageReceived = @(__original) function( _attacker, _damageHitpoints, _damageArmor )
+	{
+		__original(_attacker, _damageHitpoints, _damageArmor);
+
+		this.getActor().m.RF_DamageReceivedFromFaction.add(_attacker.getFaction(), _damageHitpoints + _damageArmor);
```

Comment:
We might want to limit these values so that overkill damage is not counted towards exp/loot. Otherwise Kill-Stealing is kind of back on the menue.
If I deal 99% of the damage to an entity and an allied billman kills them for like 30% overkill damage then they would "steal" 25% of the exp from me.

Reply:
Good point. I've fixed this now.